lib/collection/src/operations/cluster_ops.rs

```diff
@@ -196,3 +204,12 @@ pub struct AbortShardTransfer {
     pub to_peer_id: PeerId,
     pub from_peer_id: PeerId,
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema, Validate)]
+pub struct StartResharding {
+    pub peer_id: Option<PeerId>,
```

Comment:
Is this peer ID the preferred target peer for resharding? Where it would otherwise just pick a semi-random one?

Reply:
Yes.